### PR TITLE
tcld nexus docs

### PR DESCRIPTION
**Pull request by prasek**
## What does this PR do?

Adds `tcld nexus` docs

## Notes to reviewers

Auto generated from https://github.com/prasek/tcld/tree/nexus-tcld-docs

Builds on previous docs gen enhancements: https://github.com/prasek/temporal-cli/tree/nexus-docs

### CODE REVIEW

File: docs/production-deployment/cloud/tcld/nexus.mdx [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+
+Alias: `ttq`
+
+#### --unset-description
+
+Unset endpoint description.
+
+### allowed-namespace
+
+These commmands manage the allowed namespaces for a Nexus Endpoint.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/tcld/nexus.mdx:187]:
```suggestion
These commands manage the allowed namespaces for a Nexus Endpoint.
```

**Review by fairlydurable — CHANGES_REQUESTED** [review]
One typo

**Comment by prasek** [comment]
thanks @fairlydurable, addressed comments and should be good to go!

**Review by fairlydurable — APPROVED** [review]
All files read. This material aligns with existing CLI doc form.

I don't want to get in the way of the ticket. If this needs to be out there, there are no issues that would prevent this going live. I am approving this ticket.

I would strongly recommend a second pass on a follow-up ticket:

- The tags are not in alignment with our new SEO optimization (cc @napcs)
- The layout of the page can be substantially improved for readability. We can sync and I'd be happy to mock up suggestions.
- If this material is auto-generated and should not be directly edited (and instead go through the tcld team), that should be noted loudly in the file, whether with comments, metadata, or other non-user-facing element.

Also cc'ing in @josh-berry and @yuandrew since this feedback will likely affect Temporal CLI doc generation.

**Pull request merged**